jekyll/_cci2/creating-orbs.md

```diff
@@ -0,0 +1,332 @@
+</aside>
+
+#### Namespaces
+
+Namespaces are used to organize a set of orbs. Each namespace has a unique and immutable name within the registry, and each orb in a namespace has a unique name. For example, the orb ```circleci/rails``` means the rails orb in the ```circleci``` namespace may coexist in the registry with an orb called ```somenamespace/rails``` because they are in separate namespaces.
+
+Namespaces are owned by organizations. Only organization administrators can create namespaces.
+
+Organizations are, by default, limited to claiming only one namespace. This policy is designed to limit name-squatting and namespace noise. If you require more than one namespace please contact your account team at CircleCI.
+
+#### Development vs. Production Orbs
+
+Orb versions may be added to the registry either as development versions or production versions. Production versions are always a semantic version like 1.5.3; whereas development versions can be tagged with a string and are always prefixed with dev:.
+
+##### Development and Production Orb Security Profiles
+
+* Only organization administrators can publish production orbs.
+
+* Any member of an organization can publish dev orbs in namespaces.
+
+##### Development and Production Orb Retention and Mutability Characteristics
+
+* Dev orbs are mutable and expire. Anyone can overwrite any development orb who is a member of the organization that owns the namespace in which that orb is published.
+
+* Production orbs are immutable and long-lived. Once you publish a production orb at a given semantic version you may not change the content of that orb at that version. To change the content of a production orb you must publish a new version. CircleCI generally recommends using the ```orb publish increment``` and/or the ```orb publish promote``` commands in the ```circleci``` CLI when publishing orbs to production.
+
+##### Development and Production Orbs Versioning Semantics
+
+Development orbs are tagged with the format ```dev:<< your-string >>```. Production orbs are always published using the semantic versioning ("semver") scheme.
+
+In development orbs, the string label given by the user has the following restrictions:
+
+* Up to 1023 non-whitespace characters
+
+Examples of valid development orb tags:
+
+* Valid:
+  "dev:mybranch"
+  "dev:2018_09_01"
+  "dev:1.2.3-rc1"
+  "dev:myinitials/mybranch"
+  "dev:myVERYIMPORTANTbranch"
+
+* Invalid
+  "dev: 1" (No spaces allowed)
+  "1.2.3-rc1" (No leading "dev:")
+
+In production orbs you must use the form ```X.Y.Z``` where ```X``` is a "major" version, ```Y``` is a "minor" version, and ```Z``` is a "patch" version. For example, 2.4.0 implies the major version 2, minor version 4, and the patch version of 0.
+
+While not strictly enforced, CircleCI recommends when versioning your production orbs you use the standard semantic versioning convention for deciding what major, minor, and patch should mean semantically:
```

Comment:
> While not strictly enforced, CircleCI recommends when versioning your production orbs you use the standard semantic versioning convention for deciding when to increment major, minor, and patch:

Reply:
changed to` it is best practce when versioning your production orbs to use the standard semantic versioning convention for major, minor, and patch:`